app/scripts/templates/user_pages/item.html

```diff
@@ -6,9 +6,9 @@
     <img class="favicon" src="{{faviconUrl}}" />
```

Comment:
Not sure if we want to add an `alt` or `title` tag here like a good citizen. Not sure if we're always guaranteed to show the `{{extractedProviderName}}` in the .medium or .small views.